appserver/connectors/connectors-connector/src/main/java/com/sun/enterprise/connectors/connector/module/RarDetector.java

```diff
@@ -69,8 +69,16 @@
 @Service(name = RarDetector.ARCHIVE_TYPE)
 @Singleton
 public class RarDetector implements ArchiveDetector {
+    
+    /**
+     * Connector annotations for JCA 1.7
+     */
     private static final Class[] connectorAnnotations = new Class[]{
-            javax.resource.spi.Connector.class};
+            javax.resource.spi.Connector.class,
+            javax.resource.spi.ConfigProperty.class,
```

Comment:
`@ConfigProperty` is used in WARs which use Resource Adapters, I'd swap it for `@Activation`